src/grisp_connect_log_server.erl

```diff
@@ -108,3 +108,9 @@ jsonify_meta(#{meta := Meta} = Event) ->
     Optional = maps:without(maps:keys(Default), Meta),
     FilterFun = fun(Key, Value) -> jsx:is_term(#{Key => Value}) end,
     maps:put(meta, maps:merge(maps:filter(FilterFun, Optional), Default), Event).
+
+is_json_compatible(Term) ->
+    try jsx:is_term(Term)
+    catch error:_:_ ->
```

Comment:
`catch error:_ ` should work as well. We don't need to match for the stack trace.